source/scenes/home/SendAsset/Send/Send.container.tsx

```diff
@@ -245,11 +254,15 @@ const SendContainer: FC<IWalletSend> = ({ initAddress = '' }) => {
         memo,
       };
     }
-
+    
     accountController.updateTempTx(txConfig);
-
+    
     if (isExternalRequest) {
-      history.push(`/confirmTransaction?to=${txConfig.toAddress}&windowId=${windowId}`);
+      let URL = `/confirmTransaction?to=${txConfig.toAddress}&windowId=${windowId}`;
+      if (activeAsset.id === AssetType.Constellation) {
+        URL += `&chain=${StargazerChain.CONSTELLATION}`;
+      }
+      history.push(URL);
```

Comment:
Please use when possible [URLSearchParams()](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) to compose URLs.